yarn.lock

```diff
@@ -5627,7 +5627,7 @@ emojis-list@^3.0.0:
   resolved "https://registry.yarnpkg.com/emojis-list/-/emojis-list-3.0.0.tgz#5570662046ad29e2e916e71aae260abdff4f6a78"
   integrity sha512-/kyM18EfinwXZbno9FyUGeFh87KC8HRQBQGildHZbEuRyWFOmv1U10o9BBp8XVZDVNNuQKyIGIu5ZYAAXJ0V2Q==
 
-emotion-theming@^10.0.19:
+emotion-theming@^10.0.19, emotion-theming@^10.0.27:
```

Comment:
Do we still need `emotion-theming@^10.0.19` in here?

Reply:
I think 10.0.19 is required by some of the other packages, I had added 10.0.27 for direct use by us. I think it would be easier to stick with 10.0.19